<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPointDialog</class>
 <widget class="QDialog" name="AddPointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить точку на график</string>
  </property>
  <layout class="QFormLayout" name="_form_layout">
   <item row="0" column="0">
    <widget class="QLabel" name="_label_x">
     <property name="text">
      <string>Координата x</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="_box_x"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_label_y">
     <property name="text">
      <string>Координата y</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="_box_y"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="_button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_button_box</sender>
   <signal>accepted()</signal>
   <receiver>AddPointDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_button_box</sender>
   <signal>rejected()</signal>
   <receiver>AddPointDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
